### PR TITLE
Garden Events Further Improvements

### PR DESCRIPTION
**Pull request by darkruby501**
Add somes features (icons and links), fixes some bugs (sortin, disappearing items)

### CODE REVIEW

**Comment by darkruby501** [comment]
@Discordius This branch is ready for review.

And I believe I was mistaken, this branch is downstream of Ray's changes in that still open PR.

**Comment by darkruby501** [comment]
Hmm, maybe I want to improve how things are displayed to guest users, it's not what I quite intended. Or maybe I've done enough on this PR already.

File: packages/lesswrong/components/walledGarden/GardenCodesItem.tsx [hunk 1]
```diff
@@ -20,47 +40,90 @@ const styles = theme => ({
...
-        {eventFormat(gardenCode.startTime)}
-      </div>
-      <CreateIcon className={classes.icon} onClick={() => setEditing(true)}/>
+    </span>
+    <div className={classes.eventTime}>
+      {eventFormat(gardenCode.startTime)}
+    </div>
+    <div className={classes.trailingIcons}>
+      <LWTooltip title="Click to copy invite link" placement="right">
+                                                                      <CopyToClipboard text={inviteLink} onCopy={()=>flash({messageString:"Invite link copied!"})}>
```
Comment (Discordius) [inline packages/lesswrong/components/walledGarden/GardenCodesItem.tsx:111]: What happened with the indents here?

Comment (darkruby501) [reply packages/lesswrong/components/walledGarden/GardenCodesItem.tsx:111]: Whoa, huh. Someone fell asleep on tab or spacebar.

**Comment by Raemon** [comment]
(I have looked over this. I think there is a little more UI polishing of exact padding on icons, but, overall seems a net improvement and important to get merged soon)

**Review by Discordius — APPROVED** [review]

**Pull request merged**